x-pack/plugin/runtime-fields/qa/rest/build.gradle

```diff
@@ -0,0 +1,45 @@
+apply plugin: 'elasticsearch.testclusters'
+apply plugin: 'elasticsearch.standalone-rest-test'
+apply plugin: 'elasticsearch.rest-test'
+apply plugin: 'elasticsearch.rest-resources'
+
+restResources {
+  restTests {
+    includeCore '*'
+  }
+}
+
+testClusters.integTest {
+  testDistribution = 'DEFAULT'
+}
+
+integTest {
+  systemProperty 'tests.rest.blacklist',
+    [
```

Comment:
This is quite the list of exclusions. Seems a bit brittle. One issue I anticipate is future newly added tests also being incompatible with this approach for one reason or another.

Reply:
My goal is to fix the ones between here and line 36 fairly soon. At least theoretically, tests that fetch the mapping are never going to work but other sorts of tests *should* work. I *could* scan the test configuration for the command to fetch the mapping and skip those. It wouldn't be too bad and would shrink these.

Reply:
Another option is to add `skip` style property so these tests can opt themselves out of running against runtime field.

Reply:
We'd have to bring this notion of runtime fields testing up into the test framework itself though, which seems a bit awkward given how this is currently structured as a specific qa project.